.github/workflows/pr-closed.yml

```diff
@@ -1,33 +1,22 @@
 name: 🎉 PR closed
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
As you can see, when merging PR #2838, an error occurred in the workflow that adds a comment - https://github.com/material-extensions/vscode-material-icon-theme/actions/runs/12923290900/job/36040343832?pr=2838#step:2:23

This is because it uses `pull_request`, which does not give rights to add comments if the PR is made from a fork

If, for example, I make a PR from a branch from **this** repository, then `pull_request` will give the right to comment, but for forks - no

That's why we need to use `pull_request_target`.

Reply:
Yeah we have to change it here. I've just checked our history of that workflow file and we already had that in the past, because it was working already perfectly.

Reply:
This PR has more changes. I want to test it carefully before I approve this. But not sure when I'll have some time for that. Maybe in a few days. Let's see.

Reply:
Fixed it in bd82624d9897a79c55793dcc3deef91b71dabd43.

Reply:
> I want to test it carefully before I approve this

https://github.com/material-extensions/vscode-material-icon-theme/pull/2829/#issuecomment-2601962511